app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -120,13 +120,17 @@ def textual_group_endpoints
     )
   end
 
-  def textual_group_miq_custom_attributes
-    TextualGroup.new(_("Custom Attributes"), textual_miq_custom_attributes)
+  def textual_group_miq_displayable_custom_attributes
+    TextualGroup.new(_("Custom Attributes"), textual_miq_displayable_custom_attributes)
   end
 
-  def textual_miq_custom_attributes
-    attrs = @record.custom_attributes
+  def textual_miq_displayable_custom_attributes
+    attrs = @record.custom_attributes.where(
+      :section => @record.decorate.displayable_custom_attribute_sections
```

Comment:
Almost sure this has to be `@record.decorate.try(:displayable_custom_attribute_sections)` .. for all those that don't include the mixin (or don't have a decorator).